service/update.go

```diff
@@ -96,3 +100,20 @@ func (service *UpdateServiceImp) GetConceptUpdate(ctx context.Context, conceptId
 		return conceptUpdate, http.StatusOK
 	}
 }
+
+// function to get a diff from database
+func (service *UpdateServiceImp) GetConceptUpdateDiff(ctx context.Context, conceptId string, updateId string) (*model.UpdateDiff, int) {
+	conceptUpdate, err := service.ConceptsRepository.GetConceptUpdate(ctx, conceptId, updateId)
+	if err != nil {
+		logrus.Error("Unable to get concept update")
+		return nil, http.StatusInternalServerError
+	} else if conceptUpdate == nil {
+		logrus.Error("Concept update not found")
+		return nil, http.StatusNotFound
+	} else {
+		return &model.UpdateDiff{
+			ResourceId: conceptUpdate.ResourceId,
```

Comment:
Og her legger du til begrepet, slik det var før den aktuelle oppdateringen, i json-format. Regner med at det finnes en GO-implementasjon av json-patch du kan bruke for å bygge det opp fra lista du henter ut